deployment/ccip/changeset/state.go

```diff
@@ -812,7 +812,7 @@ func LoadChainState(ctx context.Context, chain deployment.Chain, addresses map[s
 	return state, nil
 }
 
-func (s CCIPOnChainState) ValidateOffRamp(chainSelector uint64) error {
+func (s CCIPOnChainState) ValidateRamp(chainSelector uint64, rampType deployment.ContractType) error {
```

Comment:
nit : not for this PR but it will probably help if we extend it to all contracts like you did for Ramps. It can take a slice of deployment.ContractType which will ensure all the types passed in param are non-empty in state